apps/io500/io500.pbs

```diff
@@ -0,0 +1,157 @@
+NSTRUCTIONS:
```

Comment:
Typo, missing 'I'
Can we provide a set a pre-defined parameter set based on system we already have tested ?
More generally speaking which parameters doesn't change and which one may vary ?
For those maybe add them as script params with default values so you don't need to edit the run script each time you want to change a value

Reply:
Corrected typo.
There are too many parameters to list (but I set some suitable defaults, should be able to run without editing).
Added a sample of tail of result summary file.